docs/topics/modules/closing_workitem.adoc

```diff
@@ -1,13 +0,0 @@
-[id="closing_workitem"]
-= Closing a work item
-
-After implementing and reviewing your code changes, update the relevant work item to track the work done.
-
-. Click the *Plan* tab and then *Board* to return to the Planner Board view.
-. Drag the work item to the *Resolved* state.
-+
-image::resolved_wi.png[Resolved work item]
-
-Resolving a work item communicates to the team that the task being tracked is complete. Once your task is reviewed and approved you can drag it to the *Closed* state.
-
-Congratulations! You have now created your first quickstart project in {osio}, made changes to your project code, committed the changes to GitHub, and published the new version of your project, and updated the work item to reflect your work.
```

Comment:
I will miss this.

Reply:
Aww. I'll add it back in, hopefully with enhancements, for the next minor PR.

Reply:
I was kidding!